cell_classification/segmentation_data_prep.py

```diff
@@ -100,7 +114,10 @@ def get_instance_masks(self, data_folder, cell_mask_key):
             np.array:
                 The instance mask
         """
-        return np.zeros([500, 500]), np.zeros([500, 500])
+        instance_mask = imread(os.path.join(data_folder, cell_mask_key + ".tiff"))
+        binary_instance_mask = erosion(instance_mask, np.ones([3, 3])) > 0
```

Comment:
This erosion doesn't work for objects that are flush with one another

```
xx = np.zeros((20, 20))
xx[1:5, 1:5] = 1
xx[5:10, 1:5] = 2
xx[1:5, 5:10] = 3
xx[5:10, 5:10] = 4
xx[10:15, 5:10] = 5
yy = erosion(xx, np.ones([3, 3]))
```

This is what we wrote for Mesmer:  https://github.com/vanvalenlab/deepcell-tf/blob/36b68e4f0a44b37067ced043d1f943d9fda5ef8f/deepcell/utils/transform_utils.py#L80